webpack.config.js

```diff
@@ -34,9 +35,12 @@ module.exports = {
   devServer: {
     port: 3000,
     open: true,
+    disableHostCheck: true,
     proxy: {
-      '/api': 'http://localhost:8080'
-    }
+      '/api/**': 'http://localhost:8080'
+    },
+    historyApiFallback: true,
+    public: 'http://v9_website.ngrok.io'
```

Comment:
Don't really like these hardcoded URLS -- there should at least be one source of truth. Let's try and remove as many as possible off the bat